app/src/main/java/com/github/libretube/db/dao/SubscriptionsFeedDao.kt

```diff
@@ -14,6 +15,9 @@ interface SubscriptionsFeedDao {
     @Insert(onConflict = OnConflictStrategy.REPLACE)
     suspend fun insertAll(feedItems: List<SubscriptionsFeedItem>)
 
+    @Update(onConflict = OnConflictStrategy.REPLACE)
```

Comment:
Isn't `REPLACE` the default behavior anyways, as it's supposed to update an existing item? I think providing that strategy is only needed for inserting items.